test.nix

```diff
@@ -97,11 +97,51 @@ let
           touch $out
         '';
 
+      depsTargetNotEmpty = pkgs.runCommand "depsTargetNotEmpty"
```

Comment:
nit: It would help if there is a small comment what these tests actually test for.

Reply:
Done!